aviary/subsystems/mass/flops_based/cargo.py

```diff
@@ -17,9 +16,10 @@ class CargoMass(om.ExplicitComponent):
     '''
 
     def initialize(self):
-        self.options.declare(
-            'aviary_options', types=AviaryValues,
-            desc='collection of Aircraft/Mission specific options')
+        add_aviary_option(self, Aircraft.CrewPayload.BAGGAGE_MASS_PER_PASSENGER,
+                          units='lbm')
+        add_aviary_option(self, Aircraft.CrewPayload.MASS_PER_PASSENGER, units='lbm')
```

Comment:
I think there was an early decision to declare options for variables that weren't likely to change during an optimization. I think we have softened that somewhat since then, and these do feel like variables that you might want to apply as uncertainty distributions. This ties into some of Ben and Eliot's current work. We should probably look at making these variables as another story.